timeseries/src/main/scala/com/criteo/cuttle/timeseries/CuttleProject.scala

```diff
@@ -86,6 +90,10 @@ class CuttleProject private[cuttle] (val name: String,
       if (paused) {
         logger.info("Pausing workflow")
         scheduler.pauseJobs(jobs.all, executor, xa)(Auth.User("Startup"))
+      } else {
+        val toPause = jobs.all.intersect(jobsToBePausedOnStartup)
+        logger.info(s"Pausing ${toPause.map(j => s"${j.name} (${j.id})").mkString(" ")}")
```

Comment:
Also maybe an `if (toPause.nonEmpty)` to prevent such logging with no jobs

Reply:
indeed